osquery/core/flags.cpp

```diff
@@ -8,85 +8,150 @@
  *
  */
 
-#include <tuple>
-
 #include <osquery/flags.h>
 
+namespace boost {
```

Comment:
Document why these are needed (in flags.h). Reason: there's not string-representation of boolean true/false casting.